config.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2018 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"os"
+
+	log "github.com/gravitational/logrus"
+	"github.com/gravitational/trace"
+)
+
+// Config is the configuration structure used everywhere in the code to pass settings and configuration
+type Config struct {
```

Comment:
You use here only one package and all structures/fields can be private.

Reply:
fixed.